abi/cpuid.go

```diff
@@ -0,0 +1,8 @@
+//go:build !amd64
+// +build !amd64
+
+package abi
+
+func cpuid(op uint32) (eax, ebx, ecx, edx uint32) {
+	panic("only implemented on amd64")
```

Comment:
Please change definition to

```go
func init() {
  cpuid = func(uint32) (uint32, uint32, uint32, uint32) { return 0, 0, 0, 0 }
}
```

and rename the file to `cpuid_other.go`

This makes dependencies a little less unhappy with us adding panics.